TF-2948 Apply new identity view

dab246 opened the pull request:
## Issue

#2948 

## Resolved

- Web:


https://github.com/user-attachments/assets/ba9c49a8-ace7-4ab5-9aeb-51092b24fd5c



- Mobile:

[demo-android.webm](https://github.com/user-attachments/assets/4159f415-9a4e-4090-8ed0-64631ebccaed)

hoangdat commented:
- [ ] it is nested scroll in this screen (both mobile/web), it makes user sometime hared to scroll, especially in mobile. So I propose to disable scroll for preview signature, fixed height for this. WDYT?
![image](https://github.com/user-attachments/assets/fde6c327-08c0-4976-b753-9ff5f1c98128)

hoangdat commented:
- [ ] `Create new profile` button is moving up when identity list loading. Should keep it pins to bottom of screen

<img width="1044" alt="image" src="https://github.com/user-attachments/assets/a77cd31f-41c4-4764-98f4-2051c4d4742d" />

dab246 commented:
> * [x]  it is nested scroll in this screen (both mobile/web), it makes user sometime hared to scroll, especially in mobile. So I propose to disable scroll for preview signature, fixed height for this. WDYT?
>
> * [x]  `Create new profile` button is moving up when identity list loading. Should keep it pins to bottom of screen
> 

Fixed



- Web:




https://github.com/user-attachments/assets/d871e3a4-a73d-4201-8287-f1f859fa3561




- Mobile:


[demo-android.webm](https://github.com/user-attachments/assets/985722d5-af33-4327-aa51-92bd066ed365)
